DaveHasteSpringer-sutton.md

```diff
@@ -0,0 +1,9 @@
+## Sutton, UK
+
+
+### Places to eat
+
```

Comment:
@DaveHasteSpringer I love trying the favorite spots for locals - any recommendations?